src/main/cms/cms_menu_imu.c

```diff
@@ -81,6 +81,29 @@ static const char * const osdTableDynNotchRangeType[] = {
 };
 #endif
 
+static void setProfileIndexString(char *profileString, int profileIndex, char *profileName)
+{
+    int charIndex = 0;
+    profileString[charIndex++] = '1' + profileIndex;
+
+#ifdef USE_PROFILE_NAMES
+    const int profileNameLen = strlen(profileName);
+
+    if (profileNameLen > 0) {
+        profileString[charIndex++] = ' ';
+        profileString[charIndex++] = '(';
+        for (int i = 0; i < profileNameLen; i++) {
```

Comment:
```
while(*profileName) {
   *p++ = toupper(*profileName++)
}

Reply:
While I agree that this is more compact and elegant, it certainly isn't as easy to understand - particularly for people with less programming experience. Since this is non-critical code I think I'd prefer to keep the simpler to understand version.